src/PlanetParameters/src/PlanetParameters.jl

```diff
@@ -13,6 +13,11 @@ using ParametersType
 @exportparameter k_Boltzmann       1.381e-23     "Boltzmann constant (m^2 kg/s^2/K)"
 @exportparameter Stefan            5.670e-8      "Stefan-Boltzmann constant (W/m^2/K^4)"
 @exportparameter astro_unit        1.4959787e11  "Astronomical unit (m)"
+@exportparameter kappa_vk          0.4           "Von Karman constant (1)"
+@exportparameter gamma_m           15.0          "Monin-Obhukov parameter (1)"
+@exportparameter gamma_h           9.0           "Monin-Obhukov parameter (1)"
+@exportparameter beta_m            4.8           "Monin-Obhukov parameter (1)"
+@exportparameter beta_h            7.8           "Monin-Obhukov parameter (1)"
```

Comment:
These parameters are not strictly planetary parameters but empirical parameters (and hence adjustable). They need to go into the parameter structure where we store adjustable parameters. (One could also argue the von Karman constant can go there too, but there is more general agreement on its value. For the von Karman constant, I’m fine with either choice.)